community/ndp/kernelextension/pom.xml

```diff
@@ -5,10 +5,10 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.neo4j</groupId>
-    <artifactId>parent</artifactId>
+    <groupId>org.neo4j.ndp</groupId>
+    <artifactId>neo4j-data-protocol-parent</artifactId>
```

Comment:
If you look at https://github.com/neo4j/neo4j/blob/2.3/community/cypher/cypher/pom.xml it depends on the `cypher-parent`. So this is not very different wrt other modules we have in the code base.